src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -66,6 +67,8 @@ public partial class QuicListenerOptions : System.Net.Quic.QuicOptions
         public int ListenBacklog { get { throw null; } set { } }
         public System.Net.IPEndPoint? ListenEndPoint { get { throw null; } set { } }
         public System.Net.Security.SslServerAuthenticationOptions? ServerAuthenticationOptions { get { throw null; } set { } }
+        public System.Net.Quic.QuicListener.ServerOptionsSelectionCallback? ServerOptionsSelectionCallback { get { throw null; } set { } }
```

Comment:
Same issue as above -- why isn't this (and the state value) on the options class?